tests/test_translator.py

```diff
@@ -277,6 +281,232 @@ def test_parse_raw_request_no_path(self):
         with self.assertRaises(ValueError):
             translator.parse_raw_request(raw_request)
 
+    def test_process_arguments_with_no_arguments(self):
+        # Arguments
+        args = argparse.Namespace(
```

Comment:
Ah, Yes! That'll surely simplify the code a bit, but still, keeping all this separate from the actual parser still feels a bit odd to me, tbh. :/